sw/link/chimera/common.ldh

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2022 ETH Zurich and University of Bologna. */
```

Comment:
Imo this entire file should live in the chimera repo, Cheshire is a dependency of Chimera!

Reply:
ok

Reply:
I totally agree